sdk/search/search/review/search.api.md

```diff
@@ -93,20 +344,208 @@ export interface IndexingResult {
     readonly succeeded: boolean;
 }
 
+// @public
+export interface KeepTokenFilter {
+    keepWords: string[];
+    lowerCaseKeepWords?: boolean;
+    name: string;
+    odatatype: "#Microsoft.Azure.Search.KeepTokenFilter";
+}
+
+// @public
+export interface KeywordMarkerTokenFilter {
+    ignoreCase?: boolean;
+    keywords: string[];
+    name: string;
+    odatatype: "#Microsoft.Azure.Search.KeywordMarkerTokenFilter";
+}
+
+// @public
+export interface KeywordTokenizer {
+    bufferSize?: number;
+    name: string;
+    odatatype: "#Microsoft.Azure.Search.KeywordTokenizer";
+}
+
+// @public
+export interface KeywordTokenizerV2 {
+    maxTokenLength?: number;
+    name: string;
+    odatatype: "#Microsoft.Azure.Search.KeywordTokenizerV2";
+}
+
+// @public
+export interface LengthTokenFilter {
+    max?: number;
+    min?: number;
+    name: string;
+    odatatype: "#Microsoft.Azure.Search.LengthTokenFilter";
+}
+
+// @public
+export interface LimitTokenFilter {
+    consumeAllTokens?: boolean;
+    maxTokenCount?: number;
+    name: string;
+    odatatype: "#Microsoft.Azure.Search.LimitTokenFilter";
+}
+
+// @public
+export interface ListIndexesOptions extends OperationOptions {
+    select?: string;
+}
+
 // @public
 export interface ListSearchResultsPageSettings {
     nextLink?: string;
     nextPageParameters?: RawSearchRequest;
 }
 
+// @public
+export interface MagnitudeScoringFunction {
+    boost: number;
+    fieldName: string;
+    interpolation?: ScoringFunctionInterpolation;
+    parameters: MagnitudeScoringParameters;
+    type: "magnitude";
```

Comment:
`name` feels more natural to me for this

Reply:
`name` instead of which property?

Reply:
instead of `type`, @bterlson ?